BREAKING.md

```diff
@@ -10,6 +10,23 @@ There are a few steps you can take to write a good change note and avoid needing
 - Provide guidance on how the change should be consumed if applicable, such as by specifying replacement APIs.
 - Consider providing code examples as part of guidance for non-trivial changes.
 
+## 0.53 Breaking changes
+- [Remove path() from RemoteFluidObjectHandle class](#Remove-path-from-RemoteFluidObjectHandle-class)
+
+### Remove `path()` from `RemoteFluidObjectHandle` class
+Remove `path()` from `RemoteFluidObjectHandle` which is part of `@fluidframework/runtime-utils` package. The alternative is to parse the handle whose url is the absolute path.
+
+```typescript
```

Comment:
This is not the correct usage of `RemoteFluidObjectHandle`. Also, `absolutePath` is deprecated so we should not suggest it be used. You should remove this example and just say `path()` was deprecated and is now removed.